testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksDelete.java

```diff
@@ -262,16 +272,23 @@ public void testDeletionVectorsAcrossAddFile(boolean partitioned)
         }
     }
 
-    // TODO: Add DELTA_LAKE_OSS group once they support creating a table with deletion vectors
-    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, DELTA_LAKE_EXCLUDE_104, DELTA_LAKE_EXCLUDE_113, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, DELTA_LAKE_EXCLUDE_104, DELTA_LAKE_EXCLUDE_113, PROFILE_SPECIFIC_TESTS})
     @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
     public void testDeletionVectorsTruncateTable()
     {
-        testDeletionVectorsDeleteAll(tableName -> onDelta().executeQuery("TRUNCATE TABLE default." + tableName));
+        testDeletionVectorsDeleteAll(tableName -> {
+            if (databricksRuntimeVersion.isPresent()) {
+                onDelta().executeQuery("TRUNCATE TABLE default." + tableName);
+            }
+            else {
+                assertThatThrownBy(() -> onDelta().executeQuery("TRUNCATE TABLE default." + tableName))
+                        .hasMessageContaining("Table does not support truncates");
+                throw new SkipException("OSS Delta Lake doesn't support truncating tables with deletion vector");
```

Comment:
Will it be supported if we upgrade Delta OSS version?

Reply:
At least, the next release candidate's tag doesn't contain the entry. https://github.com/delta-io/delta/releases/tag/v3.0.0rc1. I guess it will be supported in the future though.

Reply:
I bet it's gonna be supported in the future, but that also dissolves the myth that Delta is developed as a fully open source project. This is important to remember whenever a discussion on how to test Delta pops up.